Assets/Tests/PlayMode/ClientSessionTests.cs

```diff
@@ -117,19 +115,31 @@ public IEnumerator FindByBucketIdAndJoin()
             sessionSearchHandle.Find(ref findOptions, null, (ref SessionSearchFindCallbackInfo data) => { searchResult = data; });
 
             yield return new WaitUntil(() => searchResult != null);
```

Comment:
Not changed in scope; I wonder if WaitUntil should have a timeout here, so it doesn't roll infinitely.